webapp/src/generalSlices/intersectionSlice.tsx

```diff
@@ -23,7 +23,15 @@ export const intersectionMapLabelsLayer: SymbolLayer = {
 }
 
 export const initialState = {
-  intersections: [] as IntersectionReferenceData[],
+  intersections: [
```

Comment:
Is this dummy intersection needed?

Reply:
Yeah it is, it's the "-1" intersection, or "no intersection selected". It is used as the default across the cimms components. We use it in the configuration page to see global parameters